test/ClangImporter/clang-function-types.swift

```diff
@@ -2,11 +2,11 @@
 
 import ctypes
 
-// CHECK: f1: (@convention(c, cType: "int (*)(int)") (Swift.Int32) -> Swift.Int32)?
+// CHECK: f1: (@convention(c, cType: "size_t (*)(size_t)") (Swift.Int) -> Swift.Int)?
```

Comment:
I'm confused as to why this is still `Swift.Int` on 32-bit platforms?

Reply:
`Swift.Int` is always pointer-sized.

Reply:
Conversely, `size_t` is reliably not the natural export of `Swift.Int` because it's an unsigned type.

Reply:
Thanks for explaining. Makes sense.

Reply:
> Conversely, size_t is reliably not the natural export of Swift.Int because it's an unsigned type.

Hmm, then why don't we import `size_t` (or any unsigned integer for that matter) as `Swift.UInt`?

Reply:
It's for ergonomics. Swift uses `Int` for sizes: `Array.count` returns an `Int`, not a `UInt`. Casting for addition/subtraction would make code more verbose.